internal/data/generated_common.go

```diff
@@ -26,55 +26,62 @@ import (
 // This is a reference type, if passsed by value and callee modifies it the
 // caller will see the modification.
 //
-// Must use NewEmptyInstrumentationLibrary function to create new instances.
+// Must use NewInstrumentationLibrary function to create new instances.
 // Important: zero-initialized instance is not valid for use.
 type InstrumentationLibrary struct {
 	// Wrap OTLP otlpcommon.InstrumentationLibrary.
```

Comment:
It is worth explaining why we have a pointer-to-pointer here. We can probably say
"orig points to the pointer field contained somewhere else. We use pointer-to-pointer to be able to modify it in InitEmpty func."

Reply:
Done.